tools/src/benchmarker.cr

```diff
@@ -58,12 +58,8 @@ LANGS = [
   {lang: "python", targets: [
     {name: "sanic", repo: "channelcat/sanic"},
     {name: "japronto", repo: "squeaky-pl/japronto"},
-    # Sorry, it's still not working
-    # For debugging, try; (it will be failed)
-    # ```
-    # ./bin/benchmarker flast
-    # ```
-    # {name: "flask", repo: "pallets/flask"},
```

Comment:
flask is not working for me now. Did you try it?
```
./bin/benchmerker flask
```

Reply:
yes, with `make`

it was done in #108, except if I mess up my **working directory** :wink:

Reply:
Hmm, after merging https://github.com/tbrand/which_is_the_fastest/pull/108, it's still failing for me. But ok, will try it again. Thanks!